use-context/Exercise1/problem/.codesandbox/workspace.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
On the `use-context` folder name:

I’d like to keep the folders close to the bit names so they’re easier to find (and I’m assuming we will one day have multiple `useContext` bits).

Maybe `react-context-why-when-how` to match the [bit’s Confluence doc](https://bitovi.atlassian.net/wiki/spaces/COMPENDIUM/pages/464584811/React+Context+Why+when+and+how+to+share+values+across+an+application)?